### ISSUE TITLE
Logged out user should be able to view product listings but not edit them

### ISSUE THREAD

**Issue by camgrodgers** [camgrodgers/ProductionGraph#66]
Presently, users need to log in to view the product listings. Instead, they should be able to view the listings without the buttons for adding and editing products and dependencies.

**Comment by matthewpetela** [comment]
![Screenshot_2020-07-27 Production Graph](https://user-images.githubusercontent.com/25089614/88503799-c4b2f180-cfa0-11ea-9cd3-6e6c0e2157ae.png)
![Screenshot_2020-07-27 Production Graph(2)](https://user-images.githubusercontent.com/25089614/88503812-cd0b2c80-cfa0-11ea-939d-7d7b18d0de31.png)
![Screenshot_2020-07-27 Production Graph(1)](https://user-images.githubusercontent.com/25089614/88503826-d399a400-cfa0-11ea-992a-9b34da3d91a1.png)
![Screenshot_2020-07-27 Production Graph(3)](https://user-images.githubusercontent.com/25089614/88503833-d7c5c180-cfa0-11ea-90d4-3a5625875c0e.png)

Unauth users can now view the products and individual product pages. I edited the templates so unauth users can't see edit, add, etc buttons. Also, I changed the header so auth users see logout and unauth users see login/register.
